index.js

```diff
@@ -125,11 +125,18 @@ class Scriptable {
       return m;
     };
 
+    const globalProperties = Object.fromEntries(
+      Object.getOwnPropertyNames(global).map(
+        key => [key, global[key]],
+      ),
+    );
+    delete globalProperties.globalThis;
+    delete globalProperties.global;
```

Comment:
I think we still need to make `global` and `globalThis` available in the context. And the context should be the new value.

So instead of delete this two properties, can you add 
```
    sandbox.global = sandbox;
    sandbox.globalThis = sandbox;
```
After `sandbox.require.resolve = req => Module._resolveFilename(req, sandbox.module);`

Or make the PR editable by owner, so that I can push the changes.

Reply:
We had been fighting with the sandbox for a while before we figured out we needed this 😄 

At first we didn't specify this, but then the nodejs global of the current environment was serialized into the sandbox, which was then used as global environment within the script context, so all the things passed in the sandbox (like `sandbox.serverless`) weren't reachable from the script context anymore.

The `vm` module will automatically use the sandbox as global and globalThis if these variables are not in the sandbox already, so there's no need to set it. Maybe your code will work as well, but I'd rather let the `vm` sort these options out by itself.

Reply:
Confirmed that `globalThis` will automatically set to the context, but `global` doesn't.

I think it's ok to keep the current code unless other people report bugs. 😄